contributors/design-proposals/server-get.md

```diff
@@ -0,0 +1,170 @@
+# Expose `get` and `describe` output from the server
```

Comment:
This title is a little ambitious given that this doesn't really cover describe :)